### PR TITLE
154 https certificate

### PR DESCRIPTION
**Pull request by peterjah**

### CODE REVIEW

**Review by gregLibert — CHANGES_REQUESTED** [review]
:mag:

File: scripts/linux_install.sh [hunk 1]
```diff
@@ -3,6 +3,10 @@
 BINARY_URL="https://github.com/massalabs/thyra/releases/latest/download/thyra-server_linux"
 SCRIPT="Linux"
 
+MKCERT_URL="https://github.com/FiloSottile/mkcert/releases/download/v1.4.4/mkcert-v1.4.4-linux-amd64"
```
[the remark this answers is not in the document]
Comment (peterjah) [reply scripts/linux_install.sh:6]: > Why not using the last version: https://dl.filippo.io/mkcert/latest?for=linux/amd64 ?

It is really what we want ? Imho its safer to set a specific version that we know is working well. and only update it if needed

File: int/api/server.go [hunk 2]
```diff
@@ -31,12 +33,26 @@ func setAPIFlags(server *restapi.Server, startFlags StartServerFlags) {
...
+	configDir, _ := config.GetConfigDir()
+
 	if startFlags.TLSCertificate != "" {
 		server.TLSCertificate = flags.Filename(startFlags.TLSCertificate)
+	} else {
+		// Use default certificate
+		certFile := path.Join(configDir, "certs", "cert.pem")
+		if _, err := os.Stat(certFile); err == nil {
+			server.TLSCertificate = flags.Filename(certFile)
+		}
```
[the remark this answers is not in the document]
Comment (peterjah) [reply int/api/server.go]: bien vu!

File: scripts/linux_install.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (gregLibert) [reply scripts/linux_install.sh:6]: Yeah, you're right. But in practice, I'm not sure that we will have the capacity to update it on a regular basis.
You make the call.

[the remark this answers is not in the document]
Comment (peterjah) [reply scripts/linux_install.sh:6]: ok then lest take latest!

**Review by gregLibert — APPROVED** [review]
:clap: good job!

**Review by 0xMazout — APPROVED** [review]

**Review by ASAPSegfault — APPROVED** [review]
It works perfectly fine

**Pull request merged**